pkg/kab/resource_manager.go

```diff
@@ -52,15 +51,7 @@ func (rm *rm) Install(res v1alpha1.KabResource, backOffSettings wait.Backoff) er
 		if res.Content != "" {
 			installContent = []byte(res.Content)
 		} else {
-			if res.Path == "" {
-				return false, errors.New(fmt.Sprintf("resource %s does not specify Content OR Path to yaml for install", res.Name))
-			}
-			installContent, err = furl.Read(res.Path, "")
-			if err != nil {
-				log.Debugln("error reading", err)
-				return false, err
-			}
-
+			return false, errors.New(fmt.Sprintf("resource %s does not have Content for installation", res.Name))
```

Comment:
Is this an error condition? What if a resource entry legitimately has no content?

Reply:
the resource should just be removed then?

Reply:
or skipped as a no-op